src/cr/cube/matrix/measure.py

```diff
@@ -85,17 +87,29 @@ def pairwise_means_indices(self, alpha, only_larger):
         )
 
     def pairwise_p_vals_for_subvar(self, subvar_idx):
-        """_PairwiseSigPVals measure object for this cube-result and selected subvar"""
-        return _PairwiseSigPVals(
+        """_PairwiseSubvarSigPVals measure object for this cube-result"""
+        return _PairwiseSubvarSigPVals(
```

Comment:
Maybe consider a different name for this class, something like: `_PairwiseSigPValsForSubvar`. It's more explicit, and corresponds to the method name better (at least in my mind). The current one just sounds as a bunch of different words stacked together 😸